### DISCUSSION TITLE
odd crash when opening a certain map in osu!direct

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#16807] [Q&A] [answered]
https://osu.ppy.sh/beatmapsets/1113533

when opening the mapset above in osu!direct, the game crashes

parameters before crash:
filters: osu!standard, ranked maps only, sorted by difficulty(descending) 

[runtime.log](https://github.com/ppy/osu/files/8011579/runtime.log)
[performance-update.log](https://github.com/ppy/osu/files/8011580/performance-update.log)
[performance-input.log](https://github.com/ppy/osu/files/8011581/performance-input.log)
[performance-gamethread.update.log](https://github.com/ppy/osu/files/8011582/performance-gamethread.update.log)
[performance-gamethread.input.log](https://github.com/ppy/osu/files/8011583/performance-gamethread.input.log)
[performance-gamethread.draw.log](https://github.com/ppy/osu/files/8011584/performance-gamethread.draw.log)
[performance-gamethread.audio.log](https://github.com/ppy/osu/files/8011585/performance-gamethread.audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/8011586/performance-draw.log)
[performance-audio.log](https://github.com/ppy/osu/files/8011587/performance-audio.log)
[performance.log](https://github.com/ppy/osu/files/8011588/performance.log)
[network.log](https://github.com/ppy/osu/files/8011589/network.log)
[database.log](https://github.com/ppy/osu/files/8011590/database.log)

version: 2022.205.0

**Comment by frenzibyte** [comment, marked as answer]
Tracking in #16808
